src/components/native/Pressable.native.tsx

```diff
@@ -40,9 +38,7 @@ export const Pressable = forwardRef<View, PressableProps>(({ variants, style, ..
         <NativePressableReactNative
             {...props}
             ref={ref => {
-                const unistyles = typeof style === 'function'
-                    ? style({ pressed: false })
-                    : style
+                const unistyles = typeof style === 'function' ? style({ pressed: false }) : style
```

Comment:
why it's once in multiple lines and sometimes in single line?

Reply:
This is the default Biome's formatting for ternary, there is a discussion opened [about it](https://github.com/biomejs/biome/discussions/2388) but no away at the moment to configure it